.env.prod

```diff
@@ -0,0 +1,22 @@
+MODE=prod
+
+DB_HOST=#{DB-HOST}#
+DB_USER=#{DB-USER}#
+DB_PASSWORD=#{DB-PASS}#
+DB_NAME=#{DB-NAME}#
+
+ASSET_MAX_UPLOAD_SIZE=5242880
+ASSET_UPLOAD_PATH=./apps/backend/src/assets
+
+JWT_SECRET=DMEaoUHNWjImMi83EWEMuTEd237ZqzFXRZcKXTquDqKYBsiCpv59oFOhpC9q7l8P
+JWT_TOKEN_ALIVE=8h
+
+OAUTH2_AUTHORIZATION_URL=http://localhost:8080/realms/inheartive/protocol/openid-connect/auth
+OAUTH2_TOKEN_URL=http://localhost:8080/realms/inheartive/protocol/openid-connect/token
+OAUTH2_CALLBACK_URL=http://localhost:3333/api/_auth/callback
+OAUTH2_CLIENT_ID=nestjs
+OAUTH2_CLIENT_SECRET=O9Xfb2cpciI1tTgzpHWjWuA7tv3FJkWO
+OAUTH2_SCOPE="profile email"
+
+API_URL_PREFIX_IOS='http://localhost:3333/api'
```

Comment:
are you sure we need to wrap it in quotes ?

Reply:
Yep, good point ;)!